spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -366,6 +375,8 @@ class SpdxLicenseWithExceptionExpression(
     override fun hashCode() = license.hashCode() + 31 * exception.hashCode()
 
     override fun toString(): String = "$license $WITH $exception"
+
+    override fun getLicenseUrl() = BASE_URL + license.toString()
```

Comment:
This should simply return `license.getLicenseUrl()`.

Reply:
Done.